Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodeGraph.cpp

```diff
@@ -503,7 +503,7 @@ namespace EMStudio
         {
             // find the min and max points
             int32 minX, maxX, startY, endY;
-            if (x1 <= x2)
+            if (x1 < x2)
```

Comment:
This change is unrelated to this PR.

Reply:
@Mattt47 
You should be utilizing branches in git, especially if you want to send pull requests back to o3de.  I see you made this pull request in your `Matt47:development` branch and continued working and pushing commits to that branch.  This pull request is based on your `Matt47:development` branch so each time you push new commits there they are being added to this pull request.

We want to avoid doing this to keep PRs succinct and "on topic" and to avoid having to run AR builds too much.

Reply:
i am sorry for that 
is it okey if i revert ?

Reply:
Probably the simplest is push a new commit that restores it back the way it was.  When this PR is merged the commits can be squashed, so it's like those intermittent commits never happened.

Then in your local repository, create a new branch and continue working from there, that way you can still make commits and push them to remote branches without touching the remote `Matt47:development` branch.

Reply:
okey i will  do a simple commit to undo changes